test/mpi/configure.ac

```diff
@@ -668,8 +671,13 @@ if test "X${pac_have_cuda}" = "Xyes" ; then
         else
             cuda_LDFLAGS="-L${with_cuda}/lib"
         fi
+        AC_PATH_PROG([NVCC], [nvcc], [nvcc_not_found], [$with_cuda/bin:$PATH])
+    else
+        AC_PATH_PROG([NVCC], [nvcc], [nvcc_not_found])
```

Comment:
We are keeping it simple here by omitting the `-ccbin` in configure. It can be easily complicated.

For CI testing, we need set `NVCC="nvcc -ccbin=$CXX"` so the `.cu` files can be compiled consistently.